### DISCUSSION TITLE
以后再也不用脚本抢委托了

### DISCUSSION THREAD

**Discussion by Lanvika** [share121/inter-knot#473] [其他]
byd哥们怎么办啊，我用自动抢委托脚本给我自己抢
了个零号空洞拉力赛，现在不敢去啊呜呜呜呜
![Uploading 8a3e37972cead6a4cb705f4bbe24619b67195329.jpg…]()

**Comment by lllliufeng** [comment]
哈哈哈再发一个委托包给别人吧哈哈哈

**Comment by ProxyPhaeton** [comment]
这个委托我接了！

**Comment by warning-Kirov-reporting** [comment]
这种事，得加钱。

**Comment by XiaoFffeng** [comment]
等下一个和你一样的冤种把你的委托接了（滑稽）
